common/Cargo.toml

```diff
@@ -58,7 +57,7 @@ siphasher = "0.3"
 backtrace = { version = "0.3", optional = true }
 chrono = { version = "0.4", optional = true }
 mc-util-logger-macros = { path = "../util/logger-macros", optional = true }
-slog = { version = "2.5", features = ["dynamic-keys", "max_level_trace", "release_max_level_trace"], optional = true }
+slog = { version = "2.6", default-features = false, features = ["dynamic-keys", "max_level_trace", "release_max_level_trace"] }
```

Comment:
We should probably rename the `log` feature (either now or later) since it no longer enables log anymore (since `slog` is now enabled regardless).

Reply:
i mean, it ensures the same functionality is available that it did before, including an `std` dependency

idk i would like to not break the api of the crate right now if possible

Reply:
Yea, that makes sense. I'm mainly just pointing out that the `log` feature no longer enables logging since it will now have already been enabled. Doesn't need to be addressed right now though.